src/util/fields.py

```diff
@@ -777,8 +837,13 @@ def convert_field_to_property(
 
 
 # alias with a name related to the new API to declare property fields (company_dependent=True attribute)
-make_field_company_dependent = convert_field_to_property
+if version_gte("saas~17.5"):
+    make_field_company_dependent = _convert_field_to_company_dependent
+else:
+    make_field_company_dependent = _convert_field_to_property
 
+# retro-compatibility
+convert_field_to_property = make_field_company_dependent
```

Comment:
Extra line to make `ruff` happy.
```suggestion
convert_field_to_property = make_field_company_dependent

```